src/statmodels.jl

```diff
@@ -460,6 +460,14 @@ struct PValue
 end
 PValue(p::PValue) = p
 
+float(p::PValue) = float(p.v)
+
+for op in [Symbol("=="), :≈, :<, :≤, :>, :≥]
+    @eval begin
+        Base.$op(p::PValue, y::Real) = $op(p.v, y)
```

Comment:
Presumably you also want
```suggestion
        Base.$op(p::PValue, y::Real) = $op(p.v, y)
        Base.$op(y::Real, p::PValue) = $op(y, p.v)
        Base.$op(p1::PValue, p2::Real) = $op(p1.v, p2.v)
```
?